tests/PHPStan/Analyser/data/template-default.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+namespace TemplateDefault;
+
+use function PHPStan\Testing\assertType;
+
+/**
+ * @template T1 = true
+ * @template T2 = true
```

Comment:
I wonder what syntax is for bound templates? Something like `@template T of bool = true`?

Reply:
Yes